.github/workflows/github-actions-yaml-test.yml

```diff
@@ -0,0 +1,77 @@
+name: ORFS variables.yaml tester and linter
+
+on:
+  push:
```

Comment:
shouldn't this be pull-request, not on push(to a branch) test linting?

otherwise I don't think it will tests pull requests as only those with write access to the ORFS can create branches

The rest of the PR I think is designed to run after updates to master

Reply:
If we are catching mismatches on pull-request then I wouldn't expect we would ever need to have the docs-pr-update.  The issue would be fixed in the original PR prior to merging.